.github/workflows/auto-fix.yml

```
@@ -0,0 +1,121 @@
+      github.event.review.author_association == 'COPILOT' ||
+      github.event.review.user.login == 'github-copilot[bot]'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Extract PR details and checkout correct branch
+        id: pr
+        run: |
```

actionlint (1.7.7)

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)